Twig/DoctrineExtension.php

```diff
@@ -290,6 +290,7 @@ public function replaceQueryParameters($query, $parameters)
             '/\?|((?<!:):[a-z0-9_]+)/i',
             static function ($matches) use ($parameters, &$i) {
                 $key = substr($matches[0], 1);
+
```

Comment:
This seems to be related to https://github.com/slevomat/coding-standard/issues/635

Will confirm and see if we can fix the upstream issue